### PR TITLE
Support resultSet.wasNull() 

### PR DESCRIPTION
**Pull request by nicktorwald**

### CODE REVIEW

**Comment by nicktorwald** [comment]
it depends on #163

File: src/main/java/org/tarantool/jdbc/SQLResultSet.java [hunk 1]
```diff
@@ -295,7 +311,8 @@ public InputStream getAsciiStream(String columnLabel) throws SQLException {
 
     @Override
     public InputStream getUnicodeStream(int columnIndex) throws SQLException {
-        return new ByteArrayInputStream(getString(columnIndex).getBytes(Charset.forName("UTF-8")));
+        String string = getString(columnIndex);
+        return string == null ? null : new ByteArrayInputStream(string.getBytes(Charset.forName("UTF-16")));
```
Comment (Totktonada) [inline src/main/java/org/tarantool/jdbc/SQLResultSet.java]: Please factor out the utf-8 → utf-16 change into its own commit.

File: src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java [hunk 2]
```diff
@@ -36,8 +36,9 @@
         "CREATE TABLE test(id INT PRIMARY KEY, val VARCHAR(100))",
         "INSERT INTO test VALUES (1, 'one'), (2, 'two'), (3, 'three')",
         "CREATE TABLE test_compound(id1 INT, id2 INT, val VARCHAR(100), PRIMARY KEY (id2, id1))",
-        "CREATE TABLE test_nulls(id INT PRIMARY KEY, val VARCHAR(100))",
-        "INSERT INTO test_nulls VALUES (1, 'a'), (2, 'b'), (3, 'c'), (4, NULL), (5, NULL), (6, NULL)",
+        "CREATE TABLE test_nulls(id INT PRIMARY KEY, val VARCHAR(100), dig INTEGER)",
+        "INSERT INTO test_nulls VALUES (1, 'a', 10), (2, 'b', 20), (3, 'c', 30), " +
+            "(4, NULL, NULL), (5, NULL, NULL), (6, NULL, NULL)",
```
Comment (Totktonada) [inline src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java]: Not about your specific commit, but I think about having a test initialization and test cases in one file again and again. It is inconvenient to jump over several files to look at a simple test case. What do you think?

File: src/test/java/org/tarantool/jdbc/JdbcResultSetIT.java [hunk 3]
```diff
@@ -220,6 +220,38 @@ public void testNullsSortingDesc() throws SQLException {
         assertFalse(resultSet.next());
     }
 
+    @Test
+    void testObjectWasNullColumn() throws SQLException {
```
Comment (Totktonada) [inline src/test/java/org/tarantool/jdbc/JdbcResultSetIT.java]: I would also check null / non-null binary string in a scalar field. SCALAR is only type we don't check here as I see.

File: src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java (hunk 2 above)
Comment (nicktorwald) [reply src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java]: Agreed. We need to refactor and change our approach to writing readable tests. We can start to increase test locality moving _arrange-act-assert_ pattern within one test method or at most in one test class for the reused arrangement (@before/@after). It's better to file it as a separate issue, I think.

Comment (Totktonada) [reply src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java]: Okay, let's do.

Comment (Totktonada) [reply src/test/java/org/tarantool/jdbc/AbstractJdbcIT.java]: For the record: https://github.com/tarantool/tarantool-java/issues/188

**Pull request merged**